types/codemirror/test/addon/hint/show-hint.ts

```diff
@@ -46,3 +46,5 @@ cm.showHint({
     completeSingle: false,
     hint: asyncHintFunc,
 });
+
+cm.showHint();
```

Comment:
so can we test all?
```ts
cm.showHint();
cm.showHint({});
cm.showHint({ async: true });
```
(I could be wrong about `async` option btw, just example)

Reply:
cm.showHint({}) is not valid. The other two are already tested.